lua/tardis/interiors/base.lua

```diff
@@ -91,6 +91,8 @@ T.Exterior = {
             mat = "drmatt/tardis/mat.wav",
             mat_damaged = "jeredek/tardis/mat_damaged.wav",
             mat_fail = "drmatt/tardis/mat_fail.wav",
+            mat_fast = "p00gie/tardis/mat_fast.wav",
+            matdamaged_fast = "p00gie/tardis/matdamaged_fast.wav",
```

Comment:
Could we make this `mat_damaged_fast` to be consistent with `mat_damage`?

Reply:
Also please can you change the actual sound filename the same way!

Reply:
> Could we make this `mat_damaged_fast` to be consistent with `mat_damage`?

If you mean the sound itself to be consistent with demat effect i will do it soon(im working on my gmod video right now)

Reply:
No I mean the name of the variable in the code and the filename of the sound

Reply:
> No I mean the name of the variable in the code and the filename of the sound

I did both